mastering-plone-5/eggs1.md

```diff
@@ -2,19 +2,21 @@
 
 # Write Your Own Add-Ons to Customize Plone
 
-````{sidebar}
-Get the code! ({doc}`More info <code>`)
+````{sidebar} Get the code!
+
 Code for the beginning of this chapter:
 
-```
+```shell
 git checkout XXX
```

Comment:
I just noticed that `XXX` might not be an actual branch. Are there two branches for this page?